app/activeproject.cpp

```diff
@@ -465,31 +450,28 @@ void ActiveProject::setMapTheme( const QString &themeName )
 
 void ActiveProject::updateActiveLayer()
 {
+  QList< QgsMapLayer * > visibleLayers = getVisibleLayers();
+
+  if ( !mQgsProject || visibleLayers.isEmpty() )
```

Comment:
Checking `mQgsProject` is not really needed, as `visibleLayers` will be empty if `!mQgsProject`.
Not bad checking again, though!